drivers/sensor/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ add_subdirectory_ifdef(CONFIG_MPU6050		mpu6050)
 add_subdirectory_ifdef(CONFIG_MS5607		ms5607)
 add_subdirectory_ifdef(CONFIG_MS5837		ms5837)
 add_subdirectory_ifdef(CONFIG_OPT3001		opt3001)
+add_subdirectory_ifdef(CONFIG_HDC20X0		hdc20x0)
```

Comment:
should be alphabetically ordered, correct? Not sure it is always respected, but at least a new driver should do it.